client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -75,7 +75,7 @@ function HelpSearchResults( {
 
 	return (
 		<div>
-			<QueryInlineHelpSearch query={ searchQuery } requesting={ isSearching } />
+			<QueryInlineHelpSearch query={ searchQuery } />
 			{ renderSearchResults() }
 		</div>
```

Comment:
We can use a Fragment here `<>` instead of a `<div>`.

Reply:
Glad to update it but honestly this PR isn't taking over of this specific task. Every time that we add new commits we need to pass the tests process, which is great but sometimes it delays the workflow considerably.
Maybe, instead of adding more changes on this PR, we can create a new janitorial one.

Reply:
![image](https://user-images.githubusercontent.com/77539/86246550-8e40ad00-bb81-11ea-8e23-dc4583c5070e.png)

:'(